src/wasm/WasmModule.cpp

```diff
@@ -387,12 +388,14 @@ std::future<int32_t> WasmModule::executePthreadTask(threads::PthreadTask t)
             threadStacks.pop_back();
 
             pthreads.emplace_back([this, stackTop] {
+                int queueTimeoutMs =
+                  faabric::util::getSystemConfig().boundTimeout;
+
                 for (;;) {
                     auto logger = faabric::util::getLogger();
 
                     std::pair<std::promise<int32_t>, threads::PthreadTask>
-                      taskPair =
-                        pthreadTaskQueue.dequeue(TASK_QUEUE_TIMEOUT_MS);
```

Comment:
Having this hard-coded was confusing when you can modify all the other timeouts in the system via the config.